PEN-528 Unit tests for article body

RohitTWP opened the pull request:
Unit tests for article-body chain.
Ignored article-body feature from istanbul test coverage in order to commit my changes and create a pr.
Ignored unit tests for amp code on oembed as two of the lines are not behaving as expected.

RohitTWP committed:
PEN-528 Unit tests for article body

jgrosspietsch reviewed (changes requested):
I have a bunch of mostly semantic change suggestions. Some are more organizational.

RohitTWP committed:
code review fixes

- Changed the description for each individual describe block as suggested
- Moved all the tests in default.test to be under one describe block
- Added .skip to all OEmbed amp test cases

RohitTWP committed:
oembed shortcode fix

RohitTWP committed:
tests for Image

shk172 committed:
Merge branch 'master' into Unit-tests-for-article-body-chain
